docs/document/docs/getting-started/custom-css.md

```diff
@@ -0,0 +1,75 @@
+---
+id: custom-css
+title: Custom CSS
+sidebar_label: Custom CSS
+---
+
+## Overview
+
+**Kintone UI Component (KUC)** provides some Custom CSS properties for each component by using [CSS inheritance](#css-inheritance) and [CSS variables and custom properties](#css-variables-and-custom-properties) together. It's easy to create customizable elements. By applying CSS selectors to customize CSS custom properties, tree-based and per-instance theming are straightforward to apply.
+
+### CSS inheritance
+
+CSS inheritance lets parent and host elements propagate certain CSS properties to their descendants.
+
+### CSS variables and custom properties
+
+All CSS custom properties (`--custom-property-name`) inherit. You can use this to make your component's styles configurable from outside.
+
+KUC will publish some properties for each element, based on that users can customize CSS easily.
+
+## Usage
+
+You can set the value for `--my-background` by using the CSS element selector:
```

Comment:
```suggestion
You can set the value for `--my-background` by using the CSS selectors:
```
It seems like you can use not only element selectors but other CSS selectors in general.